docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,81 @@
+.. From the Account Settings page, select *General*, which is the default, then *Auth Token*.
+.. Under the *Auth Token* section, click on *Click to View* button to see the *KEY* value of the automatically generated token.
+. To enable Snyk code, go to *Settings* on the left panel > *Snyk Code* > *Enable Snyk Code* > click the *Disabled* bar.
+.. Save the changes when it is changed to *Enabled*.
+. Add your new secret *KEY* value to your workspace.
+    .. Login to the console https://console.redhat.com/preview/application-pipeline.
+    .. From the left menu, click *Secrets*.
+    .. Click *Add secret*.
+    .. The *Add secret* page displays options for your new secret. Specify the following:
+    ... Select Secret for *Build*.
+    ... From the *Secret type* drop-down menu, choose *Key/value secret*.
+    ... From the *Secret name* drop-down menu, select *snyk-secret*.
+    ... Under *Key/value secret*, expand *Key/value 1* section the *Key* name is automatically populated as *snyk-token*.
+    ... Paste your Snyk token into the *Upload the file with value for your key or paste its contents* field.
+    ... Click *Add secret* to save it.
+. Update your *.tekton* YAML files to specify your Snyk secret.
+.. From your GitHub repository, update the *pull-request.yaml* files in your *.tekton* directory; for example: link:https://github.com/sonam1412/devfile-sample-python-basic/blob/main/.tekton/devfile-sample-python-basic-iea1-pull-request.yaml[devfile-sample-python-basic/.tekton/devfile-sample-python-basic-rhq8-pull-request.yaml]
+.. Add new lines to the code in your YAML files to include *snyk-secret*, as shown in the following example:
++ 
+.Example:
++
+[source,yaml]
+--
+    - default: "false"
+      description: Execute the build with network isolation
+      name: hermetic
+      type: string
+    - default: ""
+      description: Build dependencies to be prefetched by Cachi2
+      name: prefetch-input
+      type: string
+    - default: "false"
+      description: Java build
+      name: java
+      type: string
+    - default: "snyk-secret"
+      description: Snyk Token Secret Name
+      name: snyk-secret
+      type: string
+--
+
+You've enabled the Snyk task for your build pipeline.
+
+== Obtaining a Red Hat Snyk token using a service account
+
+[discrete]
+=== Create an organization-level Snyk token
+
+. Register for a Snyk account or log in at https://app.snyk.io/.
+. Under *Group* in the left navigation panel, select *Red Hat*.
+. Under *Organization* in the same panel, select the name of your organization.
+. Select *Members* to identify who from your organization has *Org Admin* access. Only those members with *Org Admin* access can create a service account and token.
+. Create a service account.
+.. From the left panel, select *Settings*, then when the Settings page opens, select *Service accounts*.
+.. Name your service account, select the appropriate role from the drop-down menu, then click *Create*.
+
+IMPORTANT:
```

Comment:
lines 72-74: this isn't rendering correctly and i think it's bc you have "IMPORTANT:" and then the warning on separate lines. try this change and let's see if it looks better:

--> IMPORTANT: Snyk displays your...

also, do you want this wanring to have two paragraphs? (i see where you returned and indented at line 74.) if so, then just do 2 returns (i think!), no indent, where you want to split the copy into 2 'graphs. or put a bullet in front of each separate thought? LMK if that doesn't make sense!

Reply:
It does. I am not sure if I am doing that right because its not turning out the way as expected

Reply:
without looking, i bet i can guess bc i remember this happening to me. i suspect you just need to put a + between the two paragraphs. try that and if it doesn't work we'll figure out it tomorrow!